apps/webapp/src/components/swap/asset-out-selector.tsx

```diff
@@ -29,7 +31,7 @@ export const AssetOutSelector = ({ balances, setAssetOut, assetOut }: AssetOutSe
                 <div key={d.display} className='flex flex-col'>
                   <DialogClose>
                     <div
-                      className='grid cursor-pointer grid-cols-3 py-[10px] font-bold text-muted-foreground hover:-mx-4 hover:bg-light-brown hover:px-4'
+                      className='grid cursor-pointer break-all py-[10px] font-bold text-muted-foreground hover:-mx-4 hover:bg-light-brown hover:px-4'
```

Comment:
Truncation problems caused the delegation tokens to overflow the box. Now doing word breaking:

<img width="514" alt="Screenshot 2024-02-28 at 5 53 37 PM" src="https://github.com/penumbra-zone/web/assets/16624263/1c59a3d7-5a53-4837-82ef-59cbb7dec9ac">